bin/picca_Pk1D.py

```diff
@@ -464,6 +494,23 @@ def main(cmdargs):
     #create output dir if it does not exist
     os.makedirs(args.out_dir, exist_ok=True)
 
+    if args.nb_pixel_min is None and args.nb_pixel_frac_min is None:
+            # could make this fraction a new default, but that would probably cause trouble for SBs
+            # args.nb_pixel_frac_min = 0.13
+            args.nb_pixel_min = 75    #this is the previous default
+    elif not (args.nb_pixel_frac_min is None or args.nb_pixel_min is None):
+        print("both nb_pixel_frac_min and nb_pixel_min were set, using the latter")
+        args.nb_pixel_frac_min=None
+
+    if args.nb_pixel_masked_frac_max is None and args.nb_pixel_masked_max is None:
+            # could make this, i.e. 10% of the estimated forest length the new default
+            args.nb_pixel_masked_frac_max = 0.21
```

Comment:
Is this the default value ? why it is set differently than the values in the parser ?

Reply:
I'm setting it differently to allow both options, either setting a number of pixels or setting a fraction...